tests/conftest.py

```diff
@@ -396,3 +402,117 @@ def log_cli_level(pytestconfig):
 
     """
     return pytestconfig.getini('log_cli_level') or 'DEBUG'
+
+
+@pytest.fixture(scope="session")
+def run_io_in_background(request):
+    """
+    Run IO during the test execution
+    """
+    if config.RUN['cli_params'].get('io_in_bg'):
```

Comment:
I see you dropped the condition checking `'google_api_secret'` config value. What would happen now when this is not defined?

Reply:
Thanks @mbukatov, should be fixed now

Reply:
ok